features/fixtures/shared/scenarios/OOMScenario.m

```diff
@@ -66,11 +67,11 @@ - (void)run {
 - (void)consumeAllMemory {
     struct utsname system = {0};
     uname(&system);
-    NSLog(@"*** Device = %s", system.machine);
+    logInfo(@"*** Device = %s", system.machine);
```

Comment:
From my perspective, logs like these are too low-level to have the same prefix.  What I'd really like to do is come to a device log and be able to quickly get a medium-level context of what's going on with the e2e tests.  So (for me) these things are all really valid:
- Launching the app
- Finding reading the fixture config file
- Polling for commands
- Starting scenarios
- etc

But things that are very scenario specific I would rather leave to normal logging, as not to dilute the power of being able to focus in on an area very quickly.  Maybe we could have another standard prefix for the more detailed stuff?

Reply:
We could add a "debug" channel for stuff like this...